model/task/db.go

```diff
@@ -1675,3 +1657,16 @@ func AddHostCreateDetails(taskId, hostId string, execution int, hostCreateError
 		}})
 	return errors.Wrapf(err, "error adding details of host creation failure to task")
 }
+
+func FindActivatedStepbackTasks(projectId string) ([]Task, error) {
+	tasks, err := Find(bson.M{
+		ProjectKey:     projectId,
+		ActivatedKey:   true,
+		ActivatedByKey: evergreen.StepbackTaskActivator,
+		StatusKey:      bson.M{"$in": evergreen.TaskUncompletedStatuses},
```

Comment:
I happened to notice that TaskContainerUnallocated and TaskStatusPending are not part of TaskUncompletedStatuses. Should it be?

Reply:
Pending is a UI state, not a real status. Unallocated does seem relevant.